data/transaction/transaction_test.go

```diff
@@ -68,15 +68,15 @@ func TestTransaction_GetValue(t *testing.T) {
 	assert.Equal(t, value, tx.Value)
 }
 
-func TestTransaction_SetData(t *testing.T) {
-	t.Parallel()
-
-	data := "data"
-	tx := &transaction.Transaction{}
-	tx.SetData(data)
-
-	assert.Equal(t, data, tx.Data)
-}
+//func TestTransaction_SetData(t *testing.T) {
```

Comment:
Why was this test commented?

Reply:
this was commented before I rebuilt the capn schema and I forgot to uncomment it.